include/scale/scale_decoder_stream.hpp

```diff
@@ -75,10 +88,13 @@ namespace scale {
      * @return reference to stream
      */
     template <class F, class S>
+      requires(not std::is_reference_v<F> and not std::is_reference_v<S>)
```

Comment:
Are references allowed?
Using both `not is_reference_v` and `remove_cvref_t` is ambigous.

Reply:
legacy. removed